Brokerages/GDAX/GDAXBrokerage.cs

```diff
@@ -143,7 +144,11 @@ public override bool CancelOrder(Order order)
                 var req = new RestRequest("/orders/" + id, Method.DELETE);
                 GetAuthenticationToken(req);
                 var response = RestClient.Execute(req);
-                success.Add(response.StatusCode == System.Net.HttpStatusCode.OK);
+                success.Add(response.StatusCode == HttpStatusCode.OK);
+                if (response.StatusCode == HttpStatusCode.OK)
+                {
+                    OnOrderEvent(new OrderEvent(order, LastHeartbeatUtcTime, 0, order.Tag));
```

Comment:
Doesn't this fire an order event with the current status  -- `OrderStatus.CancelPending` ?

This line should pass the expected status:
```
OnOrderEvent(new OrderEvent(order, DateTime.UtcNow, 0, "GDAX Order Event") { Status = OrderStatus.Canceled });
```